snyk/client.py

```diff
@@ -30,35 +34,86 @@ def __init__(
         }
         self.api_post_headers = self.api_headers
         self.api_post_headers["Content-Type"] = "application/json"
+        self.tries = tries
+        self.backoff = backoff
+        self.delay = delay
+
+    def make_http_call(
```

Comment:
With an attempt to adhere to HTTP standards, how about we rename `make_http_call` into `request`?

Reply:
done!